toolkit/scripts/tools.mk

```diff
@@ -146,7 +146,7 @@ worker_chroot_deps := \
 	$(PKGGEN_DIR)/worker/create_worker_chroot.sh
 
 ifeq ($(REFRESH_WORKER_CHROOT),y)
-$(chroot_worker): $(worker_chroot_deps)
+$(chroot_worker): $(worker_chroot_deps) $(depend_REBUILD_TOOLCHAIN) $(depend_TOOLCHAIN_ARCHIVE)
```

Comment:
Possibly add a comment for `$(worker_chroot_deps)` for how it inspects the toolchain cache